src/borrower/feed/navfeed.sol

```diff
@@ -156,7 +155,7 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets, FixedPoint {
     // On borrow: the discounted future value of the asset is computed based on the loan amount and addeed to the bucket with the according maturity Date
     function _borrow(uint loan, uint amount) internal returns(uint navIncrease) {
         // ceiling check uses existing loan debt
-        require(ceiling(loan) >= safeAdd(borrowed[loan], amount), "borrow-amount-too-high");
+        require(ceiling(loan) >= amount, "borrow-amount-too-high");
```

Comment:
I am wondering if we actually need both methods:

```javascript
 function ceiling(uint loan) public view returns (uint) {
        if (borrowed[loan] > currentCeiling(loan)) {
            return 0;
        }
        return safeSub(currentCeiling(loan), borrowed[loan]);
    }

function currentCeiling(uint loan) public view returns(uint) {
        bytes32 nftID_ = nftID(loan);
        return rmul(nftValues[nftID_], ceilingRatio[risk[nftID_]]);
    }
```

We only use `ceiling` here it seems. We could just use `currentCeiling` and remove the `ceiling` function.

Reply:
I think this requires a bit more investigation whether we're not relying on either externally. I'll add a separate ticket for this.